### PR TITLE
Didi/refreshview

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

File: controls/collectionview/pull-to-refresh.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+page_title: .NET MAUI CollectionView Documentation - Pull to Refresh
+description: Learn how to achieve pull to refresh functionality in the Telerik UI for .NET MAUI CollectionView.
+position: 13
+slug: collectionview-pull-to-refresh
+tags: refreshview, .net maui, maui, collectionview, pull to refresh
+---
+
+# .NET MAUI CollectionView Pull to Refresh
+
+If the list contains items which may change after the initial load, you can allow users to refresh that list.
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:12]:
```suggestion
Some lists contain items that may change after the initial load. The _pull-to-refresh_ functionality allows the user to manually trigger an update of the list.
```

File: controls/collectionview/pull-to-refresh.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: Pull to Refresh
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:2]:
```suggestion
title: Pull-to-Refresh
```

File: controls/collectionview/pull-to-refresh.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+page_title: .NET MAUI CollectionView Documentation - Pull to Refresh
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:3]:
```suggestion
page_title: .NET MAUI CollectionView Documentation - Pull-to-Refresh
```

File: controls/collectionview/pull-to-refresh.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+description: Learn how to achieve pull to refresh functionality in the Telerik UI for .NET MAUI CollectionView.
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:4]:
```suggestion
description: Learn how to achieve a pull-to-refresh functionality in the Telerik UI for .NET MAUI CollectionView.
```

File: controls/collectionview/pull-to-refresh.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+position: 13
+slug: collectionview-pull-to-refresh
+tags: refreshview, .net maui, maui, collectionview, pull to refresh
+---
+
+# .NET MAUI CollectionView Pull to Refresh
+
+If the list contains items which may change after the initial load, you can allow users to refresh that list. 
+
+You can achieve pull to refresh functionality in Telerik .NET MAUI CollectionView by using the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/refreshview?view=net-maui-8.0" target="_blank">Microsoft .NET MAUI `RefreshView`</a>.
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:14]:
```suggestion
You can achieve a pull-to-refresh functionality in Telerik .NET MAUI CollectionView by using the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/refreshview?view=net-maui-8.0" target="_blank">Microsoft .NET MAUI `RefreshView`</a>.
```

File: controls/collectionview/pull-to-refresh.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+<snippet id='collectionview-pull-to-refresh-xaml'/>
+
+**2.** Add the following namespaces:
+
+```XAML
+xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
+```
+
+**3.** Define sample business model:
```
Comment (yordan-mitev) [inline controls/collectionview/pull-to-refresh.md:30]:
```suggestion
**3.** Define a sample business model:
```

File: controls/collectionview/pull-to-refresh.md (hunk 2 above)
Comment (didiyordanova) [reply controls/collectionview/pull-to-refresh.md:2]: we will keep the title as it is :)

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**